cadquery/cq.py

```diff
@@ -1722,6 +1744,37 @@ def radiusArc(self, endPoint, radius, forConstruction=False):
         else:
             return self.sagittaArc(endPoint, -sag, forConstruction)
 
+    def tangentArcEndpoint(self, endpoint, forConstruction=False, relative=True):
```

Comment:
Regarding the name, why not just `tangentArcPoint`? Slightly shorter and I think descriptive enough.